openshift/helpers.sh

```diff
@@ -0,0 +1,116 @@
+
+function is_set_or_fail() {
+    local name=$1
+    local value=$2
+
+    if [ -v $value ] || [ $value == "not-set" ]; then
+        echo "You have to set $name" >&2
+        exit 1
+    fi
+}
+
+function tool_is_installed() {
+# Check if given command is available on this machine
+    local cmd=$1
+
+    if ! [ -x "$(command -v $cmd)" ]; then
+        echo "Error: ${cmd} command is not available. Please install it. See README.md file for more information." >&2
+        exit 1
+    fi
+}
+
+function generate_and_deploy_config() {
+    oc process -p DEPLOYMENT_PREFIX="${DEPLOYMENT_PREFIX}" \
+    -p KEYCLOAK_URL="${KEYCLOAK_URL}" \
+    -p AWS_DEFAULT_REGION="${AWS_DEFAULT_REGION}" \
+    -f "${here}/config-template.yaml" > "${here}/config.yaml"
```

Comment:
```
    -f "${here}/config-template.yaml" > "${here}/config.yaml"
                                         ^-- SC2154: here is referenced but not assigned.
```

Reply:
I moved the definition of `here` in deploy.sh up a little bit, before sourcing of helpers.sh script, so this shouldn't be a problem now.